app/services/still_needs_help_service.rb

```diff
@@ -5,7 +5,26 @@ def self.must_show_still_needs_help_flow?(client)
 
   def self.trigger_still_needs_help_flow(client)
     client.triggered_still_needs_help_at = Time.now
-    client.tax_returns.where(status: %w[intake_ready intake_in_progress]).update(status: :file_not_filing)
+    client.tax_returns.where(status: may_need_help_statuses).update(status: :file_not_filing)
     client.save
+    send_still_need_help_notification(client)
+  end
+
+  def self.send_still_need_help_notification(client)
+    ClientMessagingService.send_system_message_to_all_opted_in_contact_methods(
+      client: client,
+      locale: client.intake.locale,
+      message: AutomatedMessage::ClosingSoon.new
+    )
+  end
+
+  def self.may_need_help_statuses
+    %w[intake_info_requested intake_in_progress intake_greeter_info_requested intake_needs_doc_help]
+  end
+
+  def self.clients_who_still_may_need_help(intake_type: nil)
```

Comment:
This is only used within a Rake task. I'd like to see it inlined into the Rake task.

If you don't like that idea, consider converting `intake_type` into a mandatory parameter, since the only call site always provides it; consider covering the method with a unit test.

Not a blocker.